modules/machineset-aws-edge-zone-public.adoc

```diff
@@ -0,0 +1,54 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_aws/installing-aws-wavelength.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="installing-with-edge-node-public_{context}"]
+= Creating edge worker nodes in public subnets (optional)
+
+AWS Local or Wavelength infrastructures exposes the Public IP address in the zone,
+allowing to ingress traffic taking advantage of lower lattency when serving applications
+closer to that zone.
+
+If your workload requires deploying the edge worker nodes in public subnets in {aws-service-name-single},
+you can modify the machine set manifests when installing a cluster.
+
++
+[IMPORTANT]
+====
+By Default OpenShift deploy the worker nodes in private subnets.
+Red Hat does not recommend placing worker nodes in subnets with Public IP attached to it.
+
+Additional security groups must be created and the rules must not be opened to the internet,
+unless you required it.
+====
+
+.Procedure
+
+. Change to the directory that contains the installation program and create the manifests:
+
+.Create the manifests
+[source,sh]
+----
+$ ./openshift-install create manifests --dir <installation_directory> 
+----
+
+. The installer manifests must be created at `openshift` and `manifests` directory. Edit the
+  machine set manifest will be created by the {aws-service-name-single} in the directory
```

Comment:
What does the WZ and LZ do here after the file is configured?

Reply:
@dfitzmau this is an optional change when the user wants to deploy a node in public subnet, instead of the default (private). The user must generate the manifests, edit the machine set manifest to set the publicIp flag.

Reply:
Thanks!